spec/support/service_spec_helper.rb

```diff
@@ -32,7 +32,15 @@ def original_url
   end
 
   def topological_url(partial_path, api_version = "v2.0")
-    "http://topology/api/topological-inventory/#{api_version}/#{partial_path}"
+    "http://topology.example.com/api/topological-inventory/#{api_version}/#{partial_path}"
```

Comment:
@syncrou do you want to use URI.join here?

Reply:
@mkanoor Yeah, that's a good idea. I'll get that change in.